server/src/test/java/org/elasticsearch/search/aggregations/metrics/TDigestStateTests.java

```diff
@@ -0,0 +1,35 @@
+package org.elasticsearch.search.aggregations.metrics;
+
+import org.elasticsearch.test.ESTestCase;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+public class TDigestStateTests extends ESTestCase {
+    @Test
```

Comment:
Doesn't really matter, but I don't think you need the test annotation here.  I believe our tests are setup to just run anything that starts with `"test"`.  (Looks like this is just a carry-over from how the TDigest library tests things)